lib/setup/setupSetup.js

```diff
@@ -366,7 +377,7 @@ function Setup(options) {
                         console.error(COLOR_RED);
                         console.error('Cannot migrate DB while js-controller is still running!');
                         console.error('Please stop ioBroker and try again. No settings have been changed.' + COLOR_RESET);
-                        callback(90);
+                        return callback(90);
```

Comment:
If not returning here, the process crashes some lines below and will print ugly errors to the cli.

Reply:
Same goes for some other parts, where I've added returning the cb, because otherwise you could configure e.g. db type `nonExistingType`, cli only printed an error but continued using the input value.

Reply:
Ok for me .. norally "callback" is "process.exit()" and thats why it is only a rare problem